enabler/src/de/schildbach/pte/ShProvider.java

```diff
@@ -164,4 +168,50 @@ protected NearbyLocationsResult nearbyStationsById(final String id, final int ma
 		uri.append("&input=").append(normalizeStationId(id));
 		return htmlNearbyStations(uri.toString());
 	}
+
+	protected static final Map<String, Style> STYLES = new HashMap<String, Style>();
+
+	static {
+		// Busse Kiel
+		putKielBusStyle("1", new Style(Style.parseColor("#7288af"), Style.WHITE));
```

Comment:
Are you aware of the available shapes? By default each signet will be "rounded" (rectangular with round corners).